packages/design-system/src/components/N8nUserStack/UserStack.vue

```diff
@@ -0,0 +1,181 @@
+<script lang="ts" setup>
+import type { IUser } from '@/types';
+import N8nAvatar from '../N8nAvatar';
+import N8nUserInfo from '../N8nUserInfo';
+import type { PropType } from 'vue';
+import { computed } from 'vue';
+
+const props = defineProps({
+	users: {
+		type: Object as PropType<{ [groupName: string]: IUser[] }>,
+		required: true,
+	},
+	currentUserEmail: {
+		type: String,
+		required: true,
+	},
+	maxAvatars: {
+		type: Number,
+		default: 2,
+		validator: (value: number) => value > 0,
+	},
+	dropdownTrigger: {
+		type: String,
+		default: 'hover',
+		validator: (value: string) => ['hover', 'click'].includes(value),
+	},
+});
+
+// Filter out empty groups
+const groups = computed(() => {
```

Comment:
Maybe this could be name `nonEmptyGroups` to make it clear that it doesn't contain empty groups. Then the comment would also be unnecessary